src/main/java/org/sefglobal/scholarx/service/IntrospectionService.java

```diff
@@ -166,4 +167,42 @@ public Profile getLoggedInUser(long id)
         }
         return mentees;
     }
+
+    /**
+     * Update {@link EnrolmentState} of Approved {@link Mentee} objects of a user as Rejected
+     * to keep only one approved {@link Mentee}
```

Comment:
Let's make the description more easy to read. For example:
`Confirm a {@link Mentor} for a specific {@link Mentee}`